schema.json

```diff
@@ -143,6 +143,15 @@
               "type": "string",
               "description": "e.g. Increased profits by 20% from 2011-2012 through viral advertising"
             }
+          },
+          "keywords": {
+            "type": "array",
+            "description": "Specify technologies or skills used, e.g. HTML, CSS, Javascript",
```

Comment:
And maybe just remove the examples from this line, to keep it consistent with the `keywords` in other sections (Skills, Interests, Projects), see https://github.com/jsonresume/resume-schema/blob/v1.0.0/schema.json.